### ISSUE TITLE
Stop pushing

### ISSUE THREAD

**Issue by rvolyar** [Charles-T-King/website#15]
I need everyone to stop pushing to the master branch before i get everything setup correctly as I can't keep track of new changes. Thanks.  I will release a workflow for this project soon

**Comment by conorgil** [comment]
Check out this workflow. We use it at work with great success:
https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

**Comment by AbhinavMadahar** [comment]
Perhaps a [fork and pull model](https://help.github.com/articles/using-pull-requests/) would be better since we can discuss the pull request before it is accepted. Also, it just seems easier to me.

**Comment by conorgil** [comment]
You still use PRs as the main code review mechanism. When a feature branch
is complete, you make a pr to get it into the develop branch.

Key is to use an approach which keeps things organized and everyone can
follow. Try one approach and see how it works

**Comment by rvolyar** [comment]
I think that PR's would kill productivity since everything is getting
reviewed. It's more for big open source projects with tons of
contributors.  The push method to me seems to fit us because we have a
small group of people working on this.

**Comment by conorgil** [comment]
PRs are definitely not just for large projects. They are pretty critical to
any project IMO. I'll send some links to good related posts tomorrow.

I would highly encourage code reviews for anything merged into master branch

**Comment by AbhinavMadahar** [comment]
@conorgil agreed. I'll try to have everyone on the frontend team review the code before accepting.

**Comment by rvolyar** [comment]
I just tried the fork/pull model and so far I like it. @conorgil when you have some time, do you mind going through the readme an installing laravel onto cloud9 like me and Abhinav described? Maybe you'll find anything we missed etc.
